src/track/create-track-package-result.ts

```diff
@@ -22,18 +22,18 @@ export async function createTrackPackageResult(
   const exceptionEvents = getExceptions(formattedEvents);
   let account: CarrierAccount | undefined;
 
-  if (shipment.carrierAccountID) {
+  if (shipment.carrierAccountId) {
```

Comment:
Based on the discussions in [slack](https://auctane.slack.com/archives/C01G1SBEC1W/p1624475161002500)